packages/libarchive.rb

```diff
@@ -10,6 +10,7 @@ class Libarchive < Package
   depends_on 'acl'
   depends_on 'attr'
```

Comment:
It is outside of this PR.  I don't want to optimize acyclic paths of dependencies to avoid causing new problem by this PR.